client/components/catalog/TagFilter.vue

```diff
@@ -32,23 +35,28 @@ import map from 'lodash/map';
 import { mapState } from 'vuex';
 
 export default {
+  data: () => ({ search: '' }),
   computed: {
     ...mapState('repositories', ['tags', 'tagFilter']),
     options() {
       return map(this.tags, it => {
         const isSelected = !!find(this.tagFilter, { id: it.id });
         return { ...it, isSelected };
       });
+    },
+    filteredTags() {
+      const { options, search } = this;
+      return search
+        ? options.filter(it => it.name.toLowerCase().includes(search.toLowerCase()))
```

Comment:
[See](https://github.com/ExtensionEngine/tailor/blob/develop/client/components/repository/Outline/index.vue#L82) how we filter activities